ccf_transaction_processor/scripts/ping_test.py

```diff
@@ -71,13 +71,17 @@ def Main() :
     try :
         config = toml.load(options.ccf_config)
     except :
-        LOG.error('unable to load ccf configuration file {0}'.format(options.ccf_config))
+        LOG.info('unable to load ccf configuration file {0}'.format(options.ccf_config))
         pass
```

Comment:
failure to load the configuration file should be an error & should cause system to exit